tests/test.js

```diff
@@ -2958,6 +2958,32 @@
                     }, resp);
                 });
 
+                asyncTest('xhr headers should work', 5, function() {
+                    mixpanel.test.set_config({
+                        xhr_headers: {
+                            'x-api-token': 'test-token',
+                            'x-api-key': 'test-key'
+                        }
+                    });
+
+                    // A meaningless request
+                    mixpanel.test.track('test', {}, function(response) {
+                        same(response, 0, "xhr returned error");
```

Comment:
This is copied from the error-handling test above, but it would probably be better not to combine xhr-headers and xhr-errors into the same test case like this; in other words (respond with status 200 and expect the response to be 1 for a successful request).